core/sql/parser/ParKeyWords.cpp

```diff
@@ -832,7 +832,6 @@ ParKeyWord ParKeyWords::keyWords_[] = {
   ParKeyWord("PROTECTION",         TOK_PROTECTION,  NONRESTOKEN_),  
   ParKeyWord("PROTOTYPE",          TOK_PROTOTYPE,   COMPAQ_|RESWORD_),
   ParKeyWord("PUBLIC",             TOK_PUBLIC,      ANS_|RESWORD_),
-  ParKeyWord("PUBLISH",            TOK_PUBLISH,     NONRESTOKEN_),
```

Comment:
sqlci/sqlci_yacc.y defines PUBLISH and UNPUBLISH as token.  May want to remove this also.